### ISSUE TITLE
mouse motion to scroll: disable inertia

### ISSUE THREAD

**Issue by belm0** [tekezo/Karabiner-Elements#1827]
First, thank you for restoring mouse-motion-to-scroll functionality.

I'd like to add an option to disable inertia. My device is a marble mouse, which already has physical inertia. So the simulated inertia is unwanted.

**Comment by belm0** [comment]
I looked at the code.  It's possibly something I can contribute.

But I think some refactoring is needed (e.g. there are unused parameters such as "momentum_minus_"), and @tekezo is modifying this code very recently, so I'll defer to him.

**Comment by tekezo** [comment]
OK, I'll add an option. Please wait for a while.

**Comment by tekezo** [comment]
I've added `options.momentum_scroll_enabled` at v12.3.2.

How to upgrade to the latest beta version:
<https://pqrs.org/osx/karabiner/document.html#check-for-updates>


You can use it by this json.
https://github.com/tekezo/Files/blob/master/karabiner-elements-extra-json/mouse_motion_to_scroll_ex.json

You can import it by copy `karabiner://karabiner/assets/complex_modifications/import?url=https://raw.githubusercontent.com/tekezo/Files/master/karabiner-elements-extra-json/mouse_motion_to_scroll_ex.json` and paste the text to web browser's location bar.

**Comment by belm0** [comment]
Thank you.  I've tried it but the behavior seems quite nonlinear.  I'll try to describe it:
  * slow mouse movement causes high speed scroll
  * quick mouse movement causes low speed scroll
  * scroll seems to retreat slightly after quick mouse motion (which gives me some sickness feeling) 

I'm using `speed` parameter set to 400%.  (By the way, could "no momentum" be a parameter instead of separate rule?)

I'm comparing this behavior to a commercial product ("Smart Scroll") which I started using after the transition from Karabiner to Karabiner-Elements.  It has a smooth 1:1 mapping between mouse motion and scroll, which works very well (though I don't like trusting a proprietary binary).
